packages/lime-proto-bmx7/files/etc/uci-defaults/bmx7-id-banner.defaults

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cat << EOF > /etc/profile
+pgrep bmx7 > /dev/null && echo " BMX7 ID: $(bmx7 -c show=status /r=1 | tail -n 1 | awk '{ print $2 }')"
```

Comment:
> This would override /etc/profile, not a good solution IMO.

This is obviously a bug and not my intention. It should be `>>` :)

> Why don't you use lime-proto-bmx7 (called when lime-config is executed) ?

Possible as well, I though uci defaults are the simplest way